### PR TITLE
Enh #82: Simply Tt menu items

### PR DESCRIPTION
**Pull request by yuriimaz**

### CODE REVIEW

File: src/editor/core/plugins/heading/menu.js [hunk 1]
```diff
@@ -2,33 +2,30 @@
...
-            attrs: {level: i}
-        })
-    }
-
-    return new DropdownSubmenu([r.makeHead1, r.makeHead2, r.makeHead3, r.makeHead4, r.makeHead5, r.makeHead6], {label: context.translate("Heading")});
+function makeHeading(context, i) {
+    return blockTypeItem(context.schema.nodes.heading, {
+        id: 'makeHeading' + i,
+        attrs: {level: i},
+        label: `Heading ${i}`,
```
Comment (luke-) [inline src/editor/core/plugins/heading/menu.js:13]: Is the menu item "Headling" is translateable?

**Review by luke- — CHANGES_REQUESTED** [review]
@yuriimaz Thanks. Can you please reduce the headings from H1-H6 to H1-H4. This is suffient.

File: src/editor/core/plugins/heading/menu.js (hunk 1 above)
Comment (yuriimaz) [reply src/editor/core/plugins/heading/menu.js]: Added translating for the menu item "Heading".

**Comment by yuriimaz** [comment]
> Can you please reduce the headings from H1-H6 to H1-H4. This is suffient.

@luke- Done

**Review by luke- — APPROVED** [review]
@yuriimaz Thanks looks good. Do we need a change in the Core for the new translation text?

**Pull request merged**